vernac/himsg.ml

```diff
@@ -632,7 +632,8 @@ let explain_refiner_cannot_generalize env sigma ty =
   pr_leconstr_env env sigma ty ++ str "."
 
 let explain_no_occurrence_found env sigma c id =
-  let c = EConstr.to_constr sigma c in
+  (* XXX: Fixme *)
+  let c = EConstr.to_constr ~abort:false sigma c in
   str "Found no subterm matching " ++ pr_lconstr_env env sigma c ++
```

Comment:
Why not use pr_leconstr here?

Reply:
Indeed, I was hesitating at first whether to make this PR "tagging-only" or to also fix some trivial ocurrences. I guess it makes sense to fix a few, so I'll update this + I will change the name of the flag.